blockchain/fundamentals/altcoins/altcoins.md

```diff
@@ -37,10 +60,13 @@ Which Bitcoin fork is 4 times as fast as Bitcoin?
 ---
 ## Revision
 
-Altcoin independent public keys can be derived from the same private key.
+
+Pick the odd one out:
```

Comment:
This one really requires knowledge of the ecosystem, rather than an understanding of what altcoins are - let's write a question about altcoins in the abstract